projects/schematics/src/dependencies.json

```diff
@@ -424,6 +424,7 @@
     "@angular/platform-browser": "^17.0.5",
     "@angular/router": "^17.0.5",
     "@ng-select/ng-select": "^12.0.4",
+    "@ngrx/store": "^17.0.1",
```

Comment:
is it still needed?
I don't see any `import @ngrx/store` within opf folder

Reply:
It is not needed. I removed that.